lib/features/drilldown/SubprocessCompatibility.js

```diff
@@ -57,11 +57,20 @@ SubprocessCompatibility.prototype._handleImport = function(definitions) {
     self._processToDiagramMap[diagram.plane.bpmnElement.id] = diagram;
   });
 
-  var newDiagrams = [];
-  definitions.diagrams.forEach(function(diagram) {
-    var createdDiagrams = self._createNewDiagrams(diagram.plane);
-    Array.prototype.push.apply(newDiagrams, createdDiagrams);
-  });
+  var newDiagrams = definitions.diagrams.reduce(
+    function(newDiagrams, diagram) {
+
+      const plane = diagram.plane;
+
+      if (!plane) {
+        return newDiagrams;
+      }
+
+      const createdDiagrams = self._createNewDiagrams(plane);
+      return newDiagrams.concat(createdDiagrams);
+    },
+    []
+  );
```

Comment:
We could replace the reduce with a filter + flatMap. In my opinion, this would make the code easier to understand:

```suggestion
  var newDiagrams = definitions.diagrams
    .filter(diagram => diagram.plane)
    .flatMap(diagram => self._createNewDiagrams(diagram.plane));
```

Reply:
I'm fine to sprinkle some functional magic in.